frontend/src/hooks/use-pagination/use-pagination.hook.ts

```diff
@@ -0,0 +1,35 @@
+import { URLSearchParamsInit } from 'common/types/types';
+import { getValidPageFromSearchParams } from 'helpers/helpers';
```

Comment:
```suggestion
import { getValidPageFromSearchParams } from './helpers/helpers';
```
this isn't a reusable helper. so you need to put in near this hook